docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -903,25 +925,86 @@ BUILD(FilterDsLib1);
     without decompression. The original index compression implementation
     decompresses the rows when they are read from disk.</para>
 
+    <para>The inplace index compression format (introduced in versions 9.6.90,
+    9.8.66, and 9.10.12 9.2.0 or later) improves compression and reduces build
+    time. These formats require an engine that supports it. In other words,
+    <emphasis role="bold">if you build an index using the lz4s or lz4shc
+    formats, you must use a platform later than 9.6.90, 9.8.66, and 9.10.12 to
+    read those indexes. </emphasis></para>
+
+    <para>If you attempt to read an index with the inplace compression format
+    on a system that does not support it, you will receive an error
+    message.</para>
+
     <para>Because the branch nodes can be searched without decompression more
     branch nodes fit into memory which can improve search performance. The lz4
     compression used for the payload is significantly faster at decompressing
-    leaf pages than the previous LZW compression.</para>
+    leaf pages than the previous LZW compression. Whether performance is
+    better with lz4hc (a high-compression variant of lz4) on the payload
+    fields depends on the access characteristics of the data and how much of
+    the index is cached in memory.</para>
 
-    <para>Whether performance is better with lz4hc (a high-compression variant
-    of lz4) on the payload fields depends on the access characteristics of the
-    data and how much of the index is cached in memory.</para>
+    <para><emphasis role="bold">Compression Levels :</emphasis></para>
 
-    <para>If you attempt to read an index with the inplace compression format
-    on a system that does not support them, you will receive an error
-    message.</para>
+    <informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec align="left" colwidth="240*"/>
+
+        <colspec colwidth="836*"/>
+
+        <tbody>
+          <row>
+            <entry><emphasis role="bold">hclevel</emphasis></entry>
+
+            <entry>An integer between 0 and 9 to specify the level of
+            compression. The default is 3. Higher levels increase compression
+            times, but may be cost-effective.</entry>
```

Comment:
Higher levels increase the compression, but also increase the compression times.

I'm not sure if the cost comment is worthwhile, but if it is it needs to be clear.

This may be cost effective depending on the length of time the data is stored, and the storage costs compared to the compute costs to build the index.